prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = "Jonathan Chiang";
+    var numberOfPets = 0
+    var likesVegetables = Boolean(true);
+    var nothing = null;
+    var never;
+    console.log("fullName: "+fullName);
+    console.log("numberOfPets: " +numberOfPets);
+    console.log("likesVegetables: " +likesVegetables);
+    console.log("nothing: " + nothing);
+    console.log("never: " + never);
```

Comment:
Here you should use a `,` comma instead of a `+` plus, because certain values, like objects and arrays, will get messed up if you combine them with your strings.